src/router/routes/live/events.ts

```diff
@@ -170,6 +172,9 @@ export class EventsController extends LiveController {
     if (!request.body.eventType) {
       return Util.standardErrorHandler(new HttpError('Event type must be provided', 400), next);
     }
+    if (!request.body.eventIcon) {
```

Comment:
I don't remember exactly, but would events like lunch or team building need an icon? Maybe instead of an error we have it default to the Penn State logo or HackPSU logo. @rsquared226?

Reply:
That makes more sense. Is there a permanent URL for one of those?

Reply:
Yeah a default seems like a good idea. In the app we have default icons for each thing (from https://materialdesignicons.com/ : a "star" for activity, "restaurant-menu" for food, and "code" for workshop). Would it be tough to do a default for each of them or should we just stick to one?

Reply:
We can definitely do a default for each one. As far as I can tell, though, the icons from materialdesigns aren't hosted as images with a resource url on the website, they're drawn from a set of svg instructions. I'm thinking it might be weird to mix the two without an indicator as to how it should be interpreted? For now, I'll probably just put the psu logo as the default.

Reply:
They do have PNGs as well I'm pretty sure, we can host them through GCP or firebase or even imgur whichever works. @stan29308 thoughts?

Reply:
The PSU logo does work for now though, we can always change it later